calibration/CTDMOG/CGINS-CTDMOG-13635__20150623.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13635,CC_ptcb1,-0.00055,
 37-13635,CC_ptcb2,0,
 37-13635,CC_ptempa0,152.1775,
-37-13635,CC_ptempa1,-0.06535585,
+37-13635,CC_ptempa1,-6.535584e-02,
 37-13635,CC_ptempa2,-4.376674e-07,
-37-13635,CC_wbotc,2.5367e-07,
+37-13635,CC_wbotc,2.536653e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT